http_client/fetch_ic/identity.ts

```diff
@@ -27,7 +27,7 @@ export function getIdentity(
 
 function getAuthorizationHeaderValue(
     headers: RequestInit['headers']
-): string | undefined | Identity {
+): string | undefined | null | Identity {
```

Comment:
I thought that instead of adding null as a return type that you were going to move the check to verify that it was an identity into this function. What happened to that?

Reply:
I will explain this to you in a call right now

Reply:
This has been explained and the code improved